diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -520,7 +536,7 @@ bool DiffDriveController::reset()
   velocity_command_subscriber_.reset();
   velocity_command_unstamped_subscriber_.reset();
 
-  received_velocity_msg_ptr_.reset();
+  received_velocity_msg_ptr_.set(nullptr);
```

Comment:
why?

Reply:
Afaik this is the only way to replicate with the RealtimeBox the nullptr reset of the shared_ptr.

Reply:
:+1: